src/test/java/api/loans/DueDateScheduledNoticesTests.java

```diff
@@ -153,10 +154,11 @@ public void checkOutSchedulesDifferentBeforeDueDateNotices()
       .withLoanNotices(Arrays.asList(
         firstBeforeDueDateNoticeConfiguration,
         secondBeforeDueDateNoticeConfiguration));
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(noticePolicy)`?

Reply:
Done.